Orange/canvas/__main__.py

```diff
@@ -341,9 +394,13 @@ def onrequest(url):
     app.fileOpenRequest.connect(onrequest)
 
     settings = QSettings()
-
     settings.setValue('startup/launch-count', settings.value('startup/launch-count', 0, int) + 1)
 
+    if settings.value("error-reporting/send-statistics", False, type=bool) \
+            and is_release:
+        UsageStatistics.set_enabled(True)
+        log.info("Enabling usage statistics tracking")
```

Comment:
I suggest moving the log line to the `UsageStatistics.set_enabled` method in `canvas-core`.
```
log.info("{} usage statistics tracking".format(
    "Enabling" if state else "Disabling"
))
```